src/main/java/pro/sky/animalshelter/model/Volunteer.java

```diff
@@ -0,0 +1,79 @@
+package pro.sky.animalshelter.model;
+
+import jakarta.persistence.*;
+
+import java.util.Objects;
+
+/**
+ * Класс, описывающий волонтера
+ */
+@Entity
+@Table(name = "volunteers")
+public class Volunteer {
+    @Id
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "chat_id")
+    private Long chatId;
+
+    @Column(name = "first_name")
+    private String firstName;
+
+    @Column(name = "is_free")
+    private boolean isFree;
```

Comment:
```suggestion
    private boolean free;
```
на is boolean-методы называют, поля/переменные без is